test/api_test.js

```diff
@@ -21,14 +21,14 @@ suite('API', () => {
     await namespaces.removeTable();
   });
 
-  test('ping', () => {
+  test.skip('ping', () => {
```

Comment:
@Hexcles Thanks, that explains why my tests were failing :) Only thing now is when I run the tests, I get a verbose output "Authorized test-client for POST access..." when I run the API tests. I was wondering if there is a way to disable this for testing only?